### ISSUE TITLE
forecastie crashed on start

### ISSUE THREAD

**Issue by kompowiec** [martykan/forecastie#112]
Hi. When in phone don't have internet not at all, it run application cause crash.

If app download in data early on 3 days it nice to have if these daty later offline

**Comment by martykan** [comment]
Works for me. Does it happen often?

**Comment by kompowiec** [comment]
Somethimes. Now i see that is it strange - when first installed application (fdroid) it crash was always and just format phone helped. Afther nothing and serveral times when added issue (when i was on bumfuck). Now i can't replace without reason (because now write that don't have internet and I didn't update).

I think that so issue closed.

**Comment by Nutomic** [comment]
I'm getting the same issue with the latest F-Droid version. Can try to get a log soon if that helps.

**Comment by martykan** [comment]
Any logs help, but git master would be ideal.

**Comment by martykan** [comment]
So I will close this if @kompowiec can't replicate this again.

**Comment by Nutomic** [comment]
I also tried to reproduce this today, but couldn't do it.

**Comment by martykan** [comment]
Okay, so I will keep this open. If this happens to anyone again, please try to get a log of the crash.

**Comment by kompowiec** [comment]
i installed syslog and still wait on appear crash.

**Comment by martykan** [comment]
OK, great.

**Comment by kompowiec** [comment]
I'm very sorry but someone steal my phone with android and must buy new... unlikely to deliver within a reasonable time (if ever succeed).

**Comment by martykan** [comment]
Oh, that sucks. There's a chance you can find it, thieves are usually not very smart and keep accounts logged on.

**Comment by kompowiec** [comment]
actual i have android 6.0 and can't replace the bug - i think that can closed.

**Comment by Nutomic** [comment]
No this crash seems to be fixed for me.

**Comment by robinpaulson** [comment]
Closing, it appears gone for now.
